uber/jinja.py

```diff
@@ -0,0 +1,61 @@
+from uber.common import *
+
+
+class JinjaEnv:
+    _env = None
+    _exportable_functions = {}
+    _filter_functions = {}
+    _template_dirs = []
+
+    @staticmethod
+    def append_template_dir(dirname):
+        """
+        Add another template directory we should search when looking up templates.
+        """
+        JinjaEnv._template_dirs.insert(0, dirname)
+
+    @staticmethod
+    def env():
```

Comment:
One thing that wasn't immediately obvious to me but which I think is the case is that the moment we call ``JinjaEnv.env()`` the first time, it pulls in all the filters such that have been defined via the decorators on this class, and that any future filters which are registered via those decorators will NOT be added to the environment.

That's all fine, because ``JinjaEnv.env()`` doesn't currently get called at import time, meaning that all plugins have been imported by the time it ever gets called.  So this shouldn't cause any problems, but you should probably add a docstring to this method explaining what I just said, because otherwise a future plugin might get into a weird situation if they did call ``JinjaEnv.env()`` for some reason at import time because they weren't explicitly warned against doing that.